action_msgs/srv/CancelGoal.srv

```diff
@@ -11,5 +11,23 @@
 # Goal info containing an ID and timestamp
 GoalInfo goal_info
 ---
+# Response codes
+# Indicates the request was accepted.
+# One or more goals have transitioned to the CANCELING state.
+# The goals_canceling list should not be empty.
+int8 RESPONSE_OK=0
+# Indicates the request was rejected.
+# No goals have transitioned to the CANCELING state.
+# The goals_canceling list should be empty.
+int8 RESPONSE_REJECTED=1
+# Indicates the requested goal ID is invalid.
+# This could be because a goal with the ID is not cancelable (e.g. terminated)
+# or there is no goal that matches the ID.
+# No goals have transitioned to the CANCELING state.
+# The goals_canceling list is empty.
+int8 RESPONSE_INVALID_GOAL_ID=2
+
+# Reponse code
+int8 response
```

Comment:
A `response` field in the response message sounds awkward. Maybe rename the flag to `status` (or similar - not sure what other messages use).

Reply:
I'll take a look around for some precedent, but I'm hesitant to use `status` so it's not confused for the goal status. The original proposal was to use `result`, which I didn't want to confuse with the goal result, but I think this is sufficiently removed from that context that it could work.

Reply:
What about `error_code`?

```
ERROR_NONE=0
ERROR_REJECTED=1
ERROR_UNKNOWN_GOAL_ID=2
ERROR_GOAL_TERMINATED=3
```

Reply:
Or `return_code`?

Reply:
> Or `return_code`?

Sounds good to me - especially with the semantic `ERROR_NONE=0`.

Reply:
Updated 1591668